src/pages/Address/AddressComp.tsx

```diff
@@ -235,11 +239,25 @@ export const AddressTransactions = ({
   addressInfo: State.Address
 }) => {
   const history = useHistory()
-
+  const isMobile = useIsMobile()
+  const searchParams = useSearchParams('layout')
+  const layout = (searchParams.layout ?? 'professional') as 'professional' | 'lite'
```

Comment:
Don't use `as` this way, it breaks the type system, because `layout` does have the potential to be passed in by users with data other than `'professional' | 'lite'`.

Most of these cases can be handled with type inference.

```suggestion
  const layout = searchParams.layout === 'professional' ? 'professional' : 'lite'
```